src/libopensc/reader-pcsc.c

```diff
@@ -2548,7 +2562,7 @@ int pcsc_use_reader(sc_context_t *ctx, void * pcsc_context_handle, void * pcsc_c
 
 	if(SCARD_S_SUCCESS == gpriv->SCardGetAttrib(card_handle,
 				SCARD_ATTR_DEVICE_SYSTEM_NAME_A, (LPBYTE)
-				reader_name, &reader_name_size)) {
+				&reader_name, &reader_name_size)) {
```

Comment:
Adding `&` here is wrong

Reply:
It is actually wrong in 2 places, at line 2543, should not have & in both.